pkg/services/ngalert/models/admin_configuration.go

```diff
@@ -22,3 +23,14 @@ func (ac *AdminConfiguration) AsSHA256() string {
 	_, _ = h.Write([]byte(fmt.Sprintf("%v", ac.Alertmanagers)))
 	return fmt.Sprintf("%x", h.Sum(nil))
 }
+
+func (ac *AdminConfiguration) Validate() error {
+	for _, u := range ac.Alertmanagers {
+		_, err := url.Parse(u)
```

Comment:
Perhaps we should also assert the `url.Scheme` to be `http(s)` here to prevent scheme-less URLs? @gotjosh

Reply:
I'm not sure we should do that, I believe the underlying component should be able to accept schemeless URLs.